Sources/BinaryJSON/UnsafePointer.swift

```diff
@@ -370,17 +364,17 @@ private extension BSON {
                 
                 var scopeLength: UInt32 = 0
                 
-                let scopeBuffer = UnsafeMutablePointer<UnsafePointer<UInt8>>.alloc(1)
+                let scopeBuffer = UnsafeMutablePointer<UnsafePointer<UInt8>>(allocatingCapacity:1)
                 
-                defer { scopeBuffer.destroy(); scopeBuffer.dealloc(1) }
+                defer { scopeBuffer.deinitialize(); /*scopeBuffer.deallocCapacity(1) */ }
```

Comment:
dealloc and deallocCapacity apparently don't apply to that data type anymore. I'm going to look into it, but I think deinitialize() might do the trick here.